python/TestHarness/util.py

```diff
@@ -43,7 +43,15 @@
                     { 'TRUE'    : '1',
                       'FALSE'   : '0'
                     }
-                  }
+    },
+
+    'libtorch' :    { 're_option' : r'#define\s+MOOSE_TORCH_ENABLED\s+(\d+)',
```

Comment:
What happened to setting this requirement based on version?

Reply:
Added!